app/pages/AboutPageOur415/AboutPage.tsx

```diff
@@ -54,8 +46,8 @@ export const AboutPage = () => {
     <>
       <Masthead title={pageData.mastHead} />
       {pageData?.twoColumnContentSections?.map(
-        (section: JSX.IntrinsicAttributes & TwoColumnContentSection) => {
-          return <TwoColumnContentSection key={section._id} {...section} />;
+        (section: JSX.IntrinsicAttributes & TwoColumnContent) => {
+          return <TwoColumnContentSection key={section.key} {...section} />;
```

Comment:
I'm curious why you change to `key` here. I also don't see it on any type declaration in this PR (though admittedly I'm ignorant of the codebase and would believe it's somewhere else.)